<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Jason A. Moggridge</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 54px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h2 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h3 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h4 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h5 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h6 {
  padding-top: 59px;
  margin-top: -59px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jmoggridge">
    <span class="fa fa-github"></span>
     
    Code
  </a>
</li>
<li>
  <a href="mailto:ja.moggridge@gmail.com">
    <span class="fa fa-envelope-o"></span>
     
    Email
  </a>
</li>
<li>
  <a href="https://twitter.com/quaxlikeaduck">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore"><strong>Jason A. Moggridge</strong></h1>

</div>


<p>B.Sc. Biochemistry (Hons), Carleton University, Ottawa ON<br />
<em>Student, researcher, biologist, programmer</em><br />
ORCID id. <a href="https://orcid.org/0000-0003-1291-5868">0000-0003-1291-5868</a></p>
<hr />
<div id="publications" class="section level2">
<h2>Publications</h2>
<ul>
<li><p>Moggridge J, Biggar K, Dawson N, &amp; Storey KB. Sensitive detection of Immunoglobulin G stability using in Real-Time Isothermal Differential Scanning Fluorimetry: determinants of protein stability for antibody-based therapeutics. <em>Technol. Cancer Res. Treat.</em>; 16(6):997-1005. (2017) PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/28602127/">28602127</a></p></li>
<li><p>Hadj-Moussa H, Moggridge JA, Luu BE, Quintero-Galvis JF, Gaitán-Espitia JD, Nespolo RF, &amp; Storey KB. The hibernating South American marsupial, <em>Dromiciops gliroides</em>, displays torpor-sensitive microRNA expression patterns. <em>Sci Rep.</em>; 6:24627. (2016). PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/27090740/">27090740</a></p></li>
</ul>
<hr />
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<div id="computational" class="section level3">
<h3>Computational</h3>
<ul>
<li>Programming &amp; scientific computing with Python and R</li>
<li>HTML documents and dashboards with Rmarkdown and Shiny</li>
<li>MS Office suite</li>
<li>Various bioinformatics tools for sequencing</li>
</ul>
</div>
<div id="wet-lab" class="section level3">
<h3>Wet lab</h3>
<ul>
<li>PCR</li>
<li>Gel electrophoresis &amp; blots</li>
<li>DNA/RNA purification</li>
<li>Protein and enzyme assays</li>
<li>Cloning</li>
</ul>
<hr />
</div>
</div>
<div id="awards-scholarships" class="section level2">
<h2>Awards / Scholarships</h2>
<ul>
<li>Master of Bioinformatics Entrance Scholarship 2020 (University of Guelph)</li>
<li>Carleton University Senate Medal 2016</li>
<li>NSERC Undergraduate student research awards (2014, 2015)</li>
<li>David A Dunton Scholarship 2015 (Carleton)<br />
</li>
<li>Clarence C. Gibson Scholarship 2014 (Carleton)<br />
</li>
<li>E.W.R. Steacie Scholarship 2013 (Carleton)<br />
</li>
<li>Dean’s Summer Research Internship 2013 (Carleton)</li>
</ul>
<hr />
</div>
<div id="projects" class="section level2">
<h2>Projects</h2>
<div id="ottawa-bike-counters-dashboard" class="section level3">
<h3><em>Ottawa bike counters dashboard</em></h3>
<p>An interactive dashboard surveying nearly 10 years of municipal bike counter data from 13 different locations in Ottawa, ON. Readers can analyze and compare various aspects of each time series including: seasonality, autoregression and decomposition; effect of weather variables on traffic; and clustering of locations by similarity. I am interested in developing a predictive model for future years using time series forecasting methods.<br />
Links to <a href="https://jmoggridge.shinyapps.io/OttBikeCounters/">dashboard</a> and <a href="https://github.com/jmoggridge/ottbike_counters">Github repository</a>.</p>
</div>
<div id="ottawa-beaches-water-quality-dashboard" class="section level3">
<h3><em>Ottawa Beaches water quality dashboard</em></h3>
<p>A dashboard exploring beach water quality data from five public beaches in Ottawa during 2014-2019. Dynamic visualizations of paired observations of <em>E. coli</em> counts and beach status. Includes analysis of <em>E.coli</em> count distributions by year, time of year, and location. The dashboard will eventually include graphs of interactions between <em>E.coli</em> counts, weather and river conditions and their predictive utility in beach water monitoring. Going forward, I hope to produce forecasting models to augment the current persistence model for closures based on these relationships. Similar systems are employed by NowCast for beaches in Ohio and elsewhere.</p>
<p>Links to <a href="https://jmoggridge.shinyapps.io/OttawaBeaches/">dashboard</a> and <a href="https://github.com/jmoggridge/ottawabeaches">Github repository</a></p>
<p><!-- **Notebooks** with details on data handling (R markdown with code):   --></p>
<p><!-- 1. [Finding & downloading data](beaches_pages/1-Download.nb.html)   --> <!-- 2. [Tidying](beaches_pages/2-Tidying.nb.html)    --></p>
<p><!-- 3. [Exploratory analysis]   --> <!-- 4. [Modeling] --> <!-- 5. [Presentation & proposal slides] --> <!--   - [**Random forests on Ottawa beaches**: a predictive water safety advisory tool for five City of Ottawa beaches] --></p>
</div>
<div id="bioinformatics-notebooks" class="section level3">
<h3><em>Bioinformatics Notebooks</em></h3>
<p>(Under construction)<br />
Python implementations of <a href="http://bioinformaticsalgorithms.com/">Bioinformatics Algorithms from Pevzner and Compeau</a> for their Coursera course and problems on Rosalind.</p>
<ul>
<li>Hidden Markov Models for Sequence Alignment:
<ul>
<li><a href="bioinfo_algos_notebooks/HMMs%20for%20sequences.html">Viterbi algorithm and outcome likelihood problem</a></li>
</ul></li>
</ul>
<!-- #### **Other notebooks/tutorials/modules**   -->
<!--  - Create a github website: A short how-to create a website similar to this on in Markdown with Rstudio & knitr, and host it on GitHub for free.     -->
<!--    [tutorial notebook](webpage.creating.html) &  [github repository](https://github.com/jmoggridge/jmoggridge.github.io)      -->
<!-- ### Bioinformatics Notebooks  -->
<!-- (coming soon) -->
<!--   1. Burrows-Wheeler transform and Manber Myers suffix array construction algorithm -->
<!--   2. Tree building: Neighbour-Joining, UPGMA -->
<!--   3. Clustering: Lloyd algorithm, k-means, hierarchical clustering -->
<!--   4. Sequence analysis -->
<!--     - Alignments: Smith-Waterman, Needleman-Wunsch -->
<!--     - De Bruijn graphs & assemble -->
<!--   5. Hidden Markov Models: -->
<!--     - Viterbi algorithm, maximum likelihood  -->
<!--     - Forward/backward algorithm, Viterbi learning, Baum-Welch learning -->
<hr />
</div>
</div>
<div id="work-experience" class="section level2">
<h2>Work experience</h2>
<ul>
<li>2017-2020 - Cook at Union Local613, Ottawa, ON</li>
<li>Fall 2015 - Teaching assistant for Biochemistry 2200 with Dr. Ken Storey at Carleton University</li>
<li>2013-2014 - Cook at Union Local613, Ottawa, ON</li>
<li>2008-2012 - Cook at the Manx Pub, Ottawa, ON</li>
<li>2006-2008 - Cook at the Whalesbone, Ottawa, ON</li>
</ul>
<hr />
</div>
<div id="about-me" class="section level2">
<h2>About me</h2>
<p>Future U of Guelph master’s student in bioinformatics (Fall 2020). Interested in next-generation sequencing and string algorithms. I really enjoy solving bioinformatics problems on <a href="http://rosalind.info/users/jasonmoggridge/">Rosalind</a>. During my undergrad at Carleton, I spent three summers doing research under the supervision of <a href="http://www.kenstoreylab.com/ken-storey/">Dr. Ken Storey</a>. My undergrad research initially focused on developing assays for protein stability using a real-time PCR instrument; in my upper years and for my Honours thesis, I investigated changes in microRNA expression during hibernation in the Monito del monte <em>Dromiciops gliroides</em> and in response to dehydration stress in the African clawed frog <em>Xenopus laevis</em>.</p>
<p>Lives in Ottawa. Enjoys cycling, cross-country skiing and exploring natural areas around the city. Favourite TV shows are <em>Nathan For You</em> and <em>Freaks and Geeks</em>. I have two cats. Feel free to challenge me to a game of chess on <a href="https://lichess.org/@/JasonMoggridge">Lichess</a> (don’t worry, I’m not very good), follow me <a href="https://www.strava.com/athletes/17492854">Strava</a> or troll me on <a href="https://twitter.com/quaxlikeaduck">twitter</a>.</p>
<hr />
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
